lib/cc/config/yaml_adapter.rb

```diff
@@ -55,6 +55,9 @@ def upconvert_legacy_yaml!
           plugins = config.fetch("plugins", {})
           plugins.each do |engine, data|
             plugins[engine] = coerce_engine(data)
+            if plugins.fetch(engine)["exclude_paths"]
+              plugins.fetch(engine)["exclude_patterns"] ||= plugins.fetch(engine).delete("exclude_paths")
```

Comment:
~~Feel free to punt on this, but I'm wondering if we've considered any kind of strategy where we log a warning that they're using a legacy key, and that they should use the new thing?~~ I see you have this! That's what I get for reviewing commit-by-commit 😜